src/views/blocks/tickets/content-description.php

```diff
@@ -16,13 +16,26 @@
  */
 
 $ticket = $this->get( 'ticket' );
+$modal  = $this->get( 'is_modal' );
+$id = 'tribe__details__content' . ( true === $modal ) ?: '__modal';
+$id .= '--' . $ticket->ID;
 
-if ( ! $ticket->show_description() ) {
+if ( ! $ticket->show_description() || empty( $ticket->description ) ) {
 	return false;
 }
 ?>
-<div
-	class="tribe-block__tickets__item__content__description"
->
+<?php if ( true === $modal && $ticket->show_description() && ! empty( $ticket->description ) ) : ?>
+		<div class="tribe-block__tickets__item__details__summary">
+			<div
```

Comment:
should these be buttons? why are we using divs for something we want to focus on?